modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -529,4 +529,17 @@ Replaced by xref:backup-restore/aggregate.adoc[`neo4j-admin backup aggregate`]
 For more information, see xref:clustering/databases.adoc#s3-seed-provider[Seed from URI].
 
 
+== Changes in Neo4j 2025.x
+
+The section covers changes to Neo4j server functionality across different areas.
+
+=== Neo4j-admin tool
+
+Neo4j 2025.01::
+
+* The `neo4j-admin database copy` command.
++
+The `--from-pagecache=<size>` option is replaced with an option: `--from-pagecache, --max-off-heap-memory=<size>`, which controls the amount of off-heap memory used for the copy operation.
```

Comment:
This sounds really confusing: I read it as "--from-pagecache is replaced by --from-pagecache, --max-off-heap-memory", as if you have to write the option like: `--from-pagecache, --max-off-heap-memory=xyz` which isn't true. You should write _ONLY_ `--max-off-heap-memory=xyz` _OR_ `--from-pagecache=xyz`.

Reply:
Yes, that's the confusing part. In the command line help, `--max-off-heap-memory` is *not* mentioned in the USAGE section, there we have only `[--from-pagecache=<size>]`. However, in the OPTIONS section, we have `--from-pagecache, --max-off-heap-memory=<size>`, which looks like I need to use both together.